_maps/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm

```diff
@@ -6,37 +6,29 @@
 /turf/open/floor/plating/asteroid/basalt/lava_land_surface,
 /area/lavaland/surface/outdoors)
 "c" = (
-/turf/closed/indestructible/abductor{
+/turf/closed/wall/mineral/abductor{
+	icon = 'icons/turf/walls.dmi';
 	icon_state = "alien20"
```

Comment:
the icon var, and for that matter icon_state as well should not be edited in maps, make a type which sets these properly

Reply:
You're making it not indestructible so use the normal icons and remove these map edits

Reply:
If you don't like the base icons then a subtype with the other icons would also be fine, you may be forced to convert it into a smoothed icon set though.